server/bundler/loader.js

```diff
@@ -92,6 +99,16 @@ function requireTemplate( module ) {
 	return 'require( ' + JSON.stringify( module ) + ' )();\n';
 }
 
+function singleEnsure( chunkName ) {
+	var result = [
+		'case ' + JSON.stringify( chunkName ) + ':',
+		'	return require.ensure([], function() {}, ' + JSON.stringify( chunkName ) + ' );',
```

Comment:
Yeah, and the error handling uses the `?retry` thing, which we cannot do on preloading.